src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -930,7 +934,7 @@ export class SearchSource {
     //       },
     //     ],
     //   },
-    // };
+    // });
```

Comment:
Adding a neat hint, copyright to this: @nreese 
```suggestion
    // });
    // Alternatively you could also add this query via "Edit as Query DSL", then it needs no code to be changed
```

Reply:
Yes, good tip! I'll add this too.